runner/config/config.go

```diff
@@ -3,4 +3,7 @@ package config
 const (
 	// BufSize is the size of buffers passed in to sockets
 	BufSize = 8192
+	// ConnSockBufSize is the size of the socket used for
+	// to attach to the container
+	ConnSockBufSize = 32768
```

Comment:
Why doesn't this use the constant defined in config.h?  I guess same goes for BufSize.

Reply:
Oops I guess this is because one is Go and the other is C